Gemfile

```diff
@@ -1,6 +1,10 @@
 source 'https://rubygems.org'
 
-gem 'hydra', git: 'https://github.com/projecthydra/hydra.git'
+gem 'hydra-head', '~> 10.1'
+gem 'active-fedora', '~> 11.0'
+gem 'active_fedora-datastreams'
```

Comment:
`ActiveFedora::Datastream` and its descendants (`OmDatastream`, `NogogiriDatastream`) were pulled out of `active-fedora` into `active_fedora-datastreams`. We can't do MODS without it.

I only included `curation_concerns` because I thought it pulled in other stuff we needed. I'll comment it out and run the tests to see what happens.